src/Util.cpp

```diff
@@ -169,6 +215,29 @@ get_level_1_files(const std::string& dir,
   get_cache_files_internal(dir, 1, progress_receiver, files);
 }
 
+std::string
+get_path_in_cache(nonstd::string_view name, nonstd::string_view suffix)
```

Comment:
When refactoring and modernizing the code I'm generally trying to remove dependencies to global state, e.g. passing the config explicitly to utility functions. Do you think that this would make sense here as well?